cmd/kube-apiserver/app/server.go

```diff
@@ -133,6 +134,8 @@ cluster's shared state through which all other components interact.`,
 		},
 	}
 
+	cmd.AddCommand(insecurereadyz.NewInsecureReadyzCommand())
```

Comment:
Is there a benefit to attaching this to the kube-apiserver binary as opposed to making a new binary?